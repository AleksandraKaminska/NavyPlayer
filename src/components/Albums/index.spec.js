import React from 'react'
import { Albums } from './index'
import renderer from 'react-test-renderer'

jest.mock('./albumSongs', () => 'AlbumSongs')

describe('Albums', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Albums albums={[]} />)
    expect(tree).toMatchSnapshot()
  })
})
