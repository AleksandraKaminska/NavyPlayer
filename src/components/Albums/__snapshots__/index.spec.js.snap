// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Albums renders correctly 1`] = `
<section
  className="albums"
>
  <h2>
    Albums
  </h2>
  <div
    className="slider-frame"
  >
    <div
      className="btn prev"
      onClick={[Function]}
    >
      <i
        className="fas fa-chevron-left"
      />
    </div>
    <div
      className="btn next"
      onClick={[Function]}
    >
      <i
        className="fas fa-chevron-right"
      />
    </div>
    <div
      className="slider-container"
    />
  </div>
  <div
    className="close"
    onClick={[Function]}
  >
    <i
      aria-hidden="true"
      className="fa fa-arrow-right"
    />
  </div>
  <AlbumSongs
    songsRef={
      Object {
        "current": null,
      }
    }
  />
</section>
`;
